app/controllers/account/admin/user_controller.rb

```diff
@@ -1,6 +1,24 @@
 class Account::Admin::UserController < Account::Admin::AdminApplicationController
-  def index
-    @users = User.all
+  def users
+    @users = User.where.not(confirmed_at: nil)
```

Comment:
should be non admin users, and add scope for confirmed if devise doesn't include it